src/client/src/css/jobListing.css

```diff
@@ -253,53 +190,173 @@
   font-size: large;
 }
 
-@media all and (max-width: 690px) {
-  .jobPosts_Container {
+@media all and (max-width: 905px) {
+  .jobPosts {
+    border: 2px #19718d solid;
+    width: 85%;
+    height: 570px;
+    border-radius: 15px;
+    margin: auto;
+    margin-top: 30px;
+    display: inline-block;
+  }
+
+  .jobPosts .AlertJobListing{
+    display: none;
+  }
+  .jobPosts .YetAppliedJobs{
+    display: none;
+  }
+
+  .jobPosts .jobs .jobPost .selectButton {
+
+    margin: 20px;
+
+  }
+  
+  .jobPosts .jobs .jobPost .linkButton {
+   
+    margin: 20px;
+    
+  }
+  .jobPosts .jobs .jobPost .col-md-6{
+    margin: 20px;
+  }
+}
+
+
+@media all and (max-width: 610px) {
+
+  .jobFilter {
+    margin-bottom: 20px;
+    margin-top: 10px;
+    padding: 20px;
+    background-color: #f0f0f0;
+  }
+  
+  .jobFilter form {
     display: flex;
     flex-direction: column;
+    justify-content: space-between;
     align-items: center;
+    width: 100%;
   }
-  .jobPosts .jobs .jobPost {
-    width: 273%;
+  
+  .jobFilter label {
+    margin-right: 10px;
+    font-size: large;
   }
   .jobPosts {
-    width: 270px;
+    
+    width: 85%;
+    height: 570px;
+    border-radius: 15px;
+    margin: auto;
+    display: inline-block;
+    margin-top: 150px;
   }
-  .jobPost .col-md-6 {
-    position: relative;
-    float: right;
+  .jobPosts .jobPost{
+    min-height: 500px;
   }
-  .preferences {
-    display: flex;
-    width: 300px;
-    margin-right: 125px;
-    height: 350px;
+
+  .jobPosts .AlertJobListing{
+    display: none;
+  }
+  .jobPosts .YetAppliedJobs{
+    display: none;
+  }
+
+  .jobPosts .jobs .jobPost .selectButton {
+
+    margin: 20px;
+    position: absolute;
+    right: 50px;
+    top: 280px;
+
+  }
+  
+  .jobPosts .jobs .jobPost .linkButton {
+   
+    margin: 20px;
+    position: absolute;
+    right: 50px;
+    top: 350px;
+    
+  }
+  .jobPosts .jobs .jobPost .col-md-6{
+    margin: 20px;
   }
 }
 
-@media all and (max-width: 280px) {
-  .Container {
+
+
+@media all and (max-width: 610px) {
+
+  .logo{
+    display: none;
+  }
+
+  .jobFilter {
+    margin-bottom: 20px;
+    margin-top: 10px;
+    padding: 20px;
+    background-color: #f0f0f0;
+  }
```

Comment:
There are large vertical margins for small screens.![image](https://user-images.githubusercontent.com/78051064/230804324-d23bcc76-6d12-4446-857b-96f20a7d6667.png)